setenv.sh

```diff
@@ -24,10 +25,10 @@ export TORQPROCESSES=${KDBAPPCONFIG}/process.csv
 
 touch $KDBLOG/torqsslcert.txt
 if [ -z "${SSL_CA_CERT_FILE}" ]; then
-	mkdir ${PWD}/certs
-	curl -s  https://curl.haxx.se/ca/cacert.pm > ${PWD}/certs/cabundle.pem
-	echo "`date`    The SSL securiity certificate has been downloaded to ${PWD}/certs/cabundle.pem" </dev/null >>$KDBLOG/torqsslcert.txt 
-	export SSL_CA_CERT_FILE=${PWD}/certs/cabundle.pem
+	mkdir ${TORQHOME}/certs
```

Comment:
are these changes related to monit? seems like a separate thing (which is probably a worthwhile change, but if it's not related to monit it probably shouldn't be in monit PR)

Reply:
They are related to monit. If I don't change that `${PWD}` with `${TORQHOME}` monit won't be able to start the processes